tests/system/providers/google/cloud/video_intelligence/example_video_intelligence.py

```diff
@@ -0,0 +1,163 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""
+Example Airflow DAG that demonstrates operators for the Google Cloud Video Intelligence service in the Google
+Cloud Platform.
+
+This DAG relies on the following OS environment variables:
+
+* BUCKET_NAME - Google Cloud Storage bucket where the file exists.
+"""
+import os
+from datetime import datetime
+
+from google.api_core.retry import Retry
+
+from airflow import models
+from airflow.models.baseoperator import chain
+from airflow.operators.bash import BashOperator
+from airflow.providers.google.cloud.operators.gcs import GCSCreateBucketOperator, GCSDeleteBucketOperator
+from airflow.providers.google.cloud.operators.video_intelligence import (
+    CloudVideoIntelligenceDetectVideoExplicitContentOperator,
+    CloudVideoIntelligenceDetectVideoLabelsOperator,
+    CloudVideoIntelligenceDetectVideoShotsOperator,
+)
+from airflow.providers.google.cloud.transfers.gcs_to_gcs import GCSToGCSOperator
+from airflow.utils.trigger_rule import TriggerRule
+
+ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID")
+
+DAG_ID = "example_gcp_video_intelligence"
+
+BUCKET_NAME_SRC = "cloud-samples-data"
```

Comment:
Is it public repository with the sample data? Can you add comment explaining it above - for the future readers ;)

Reply:
yes, it's public bucket

Reply:
done